netlify.toml

```diff
@@ -0,0 +1,5 @@
+[[redirects]]
+  from = "/*"
+  to = "https://docs.openzeppelin.com/sdk"
```

Comment:
Can we add a `:splat`, assuming the names of the pages will be the same? We have several references to specific pages from the outside to the docsite.

Reply:
I want to try something like this:
```
[[redirects]]                                                                                                                                                 
  from = "/*.html"                                                                                                                                      
  to = "https://docs.openzeppelin.com/sdk/2.5/:splat"                                                                                                         
  status = 301                                                                                                                                                
  force = true                                                                                                                                                
```
But the current site has 2.4, and the new one has 2.5. In addition to that, I think it could lead to more 404, like with https://zeppelinos-docs.netlify.com/docs/start.html which has no equivalent on the new site.
I can play a little more, and propose a new change on monday morning before the announcement.